controllers/internalrequest/adapter.go

```diff
@@ -101,6 +145,16 @@ func (a *Adapter) EnsureStatusIsTracked() (reconciler.OperationResult, error) {
 	return reconciler.ContinueProcessing()
 }
 
+// getDefaultConfig creates and returns a Config resource with default values.
```

Comment:
the namespace isn't a default value, it is the value passed in

Reply:
Fixed.